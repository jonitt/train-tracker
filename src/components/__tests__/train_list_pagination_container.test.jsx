import { shallow } from "enzyme";
import React from "react";
import TrainListPaginationContainer from "../train_list_pagination/train_list_pagination_container.jsx";

describe("TrainListPaginationContainer", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<TrainListPaginationContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
