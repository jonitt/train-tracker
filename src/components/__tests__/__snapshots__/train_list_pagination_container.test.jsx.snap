// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TrainListPaginationContainer renders correctly 1`] = `ShallowWrapper {}`;
